.github/workflows/auto-updates.yaml

```diff
@@ -477,10 +477,17 @@ jobs:
       with:
         path: config
 
+    - name: Set up Go 1.15.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.x
+
     - name: Copy Actions
       shell: bash
       run: |
-        mkdir -p "${GITHUB_WORKSPACE}/main/.github/workflows/"
+        go get github.com/bronze1man/yaml2json
```

Comment:
I think you want to move this to a step before you checkout the target repo like https://github.com/knative-sandbox/knobots/pull/64/files#diff-6ee5036a4531601a2456975ffb9959057d15ff5318f066d51a0309925ee562b9R147 to above making a PR with that dep in the pending PR

Reply:
None of the repos are in the root directory, so this should just install stuff in the `$GOROOT` and not mess with any `go.mod` file.

The PR later has a `path: main`:

https://github.com/knative-sandbox/knobots/blob/main/.github/workflows/auto-updates.yaml#L504